src/Jackett.Common/Definitions/luckpt.yml

```diff
@@ -0,0 +1,185 @@
+---
+id: luckpt
+name: LuckPT
+description: "LuckPT is a CHINESE Private Torrent Tracker for MOVIES / TV / GENERAL"
+language: zh-CN
+type: private
+encoding: UTF-8
+links:
+  - https://pt.luckpt.de/
+
+caps:
+  categorymappings:
+    - {id: 401, cat: Movies, desc: "电影 / Movies"}
+    - {id: 402, cat: TV, desc: "电视剧 / TV Series"}
+    - {id: 405, cat: TV/Anime, desc: "动漫 / Animations"}
+    - {id: 406, cat: Audio/Video, desc: "演唱会、MV / Music Videos"}
+    - {id: 408, cat: Audio, desc: "音乐 / Music"}
+    - {id: 409, cat: Other, desc: "其他 / Other"}
+    - {id: 410, cat: TV, desc: "综艺 / TV Shows"}
+    - {id: 411, cat: TV/Documentary, desc: "纪录片 / Documentary"}
+    - {id: 412, cat: TV/Sport, desc: "体育 / Sports"}
+    - {id: 413, cat: TV/Other, desc: "短剧 / Playlet"}
+
+  modes:
+    search: [q]
+    tv-search: [q, season, ep, imdbid, doubanid]
+    movie-search: [q, imdbid, doubanid]
+    music-search: [q]
+
+settings:
+  - name: cookie

```

Comment:
```suggestion
    # using cookie method because site does a JS call to API/Challenge via JQuery to load response parm required for takelogin.php
    - name: cookie
```